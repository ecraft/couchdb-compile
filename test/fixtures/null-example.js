module.exports = {
  _id: 'null-example',
  'null-field': null
}
