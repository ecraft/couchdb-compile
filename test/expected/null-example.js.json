{
  "_id": "null-example",
  "null-field": null
}
